rfcs/0018-glides-trainsheet-events.md

```diff
@@ -140,69 +105,125 @@ In order to reduce event duplication, a Trip Key is used to identify both added
+- `tripUpdates` (array of ([TripUpdated](#TripUpdated)|[TripAdded](#TripAdded))): The list of one or more trips that have new information.
 
-### Trip updated
-This indicates that a trip was updated in some fashion:  consist, operators, departure time. This includes updates which confirm an arrival or departure time. Fields which are not present are not considered to be updated.
+#### TripUpdated
+This indicates that a trip was updated in some fashion:  consist, operators, departure time. The trip is either a scheduled trip, or an added trip that has already appeared in the events stream. Fields which are not present are not considered to be updated.
 
-Event type: `com.mbta.ctd.glides.trip_updated.v1`
-Fields in the event:
-- `author` (Author): the inspector adding the trip.
+Fields in the object:
+- `type` (string): `"updated"|"added"`. Determines whether this is a `TripUpdated` object or a `TripAdded` object. If it's `"added"`, then it's a `TripAdded` object, see below. Subsequent updates to previously-added trips have `type` `"updated"`.
 - `tripKey` (Trip Key): which trip is being updated.
 - `comment` (string, optional): free text information about the trip.
 - `startLocation` (Location, optional): the new destination of the train.
 - `endLocation` (Location, optional): the new destination of the train.
 - `startTime` (Time, optional): if present, the new time that the train is expected to depart `startLocation` (or the existing `startLocation` of the trip).
 - `endTime` (Time, optional): if present, the new time that the train is expected to arrive at `endLocation` (or the existing `endLocation` of the trip).
-- `consist` (array of Delta Car, optional): the cars assigned to perform this trip. If a car is `"cleared"`, then no car is currently assigned to that position in the consist. If a car is `"unmodified"` then the car is the value from the most-recent [Trip updated](#Trip-updated) event. The front car is listed first.
-- `scheduledOperators`: (array of Operator, optional): if present, the operators who were scheduled to operate this trip.
-- `operators` (array of Delta Operator, optional): the list of operators assigned to perform this trip. If an operator is `"cleared"`, then no operator is assigned to the car in that position in the consist. If the operator is `"unmodified"` then the operator is the value from the schedule or the most-recent [Trip updated](#Trip-updated) event. The operator of the front car is listed first.
-- `dropped`: (Dropped Reason or `false`, optional): if `false`, the trip is not dropped (and restored if previously dropped). If a [Dropped Reason](#dropped-reason), the reason the trip was dropped.
+- `cars` (array of [Car](#Car), optional): array of length 1 or 2, containing the car numbers and operators for each car in the train assigned to the trip. If absent, there are no changes to any cars. If present, the length of the array is the length of the train. In a two car train, the front car is listed first.
```

Comment:
I ended up keeping the `cars` name to maintain symmetry with the `scheduledCars` name. `scheduledConsist` doesn't work partially because it's awkwardly long, but mostly because the data in `scheduledCars` is less "consist"-like when it's not referring to actual vehicles like `cars[0].label` does.